spec/dummy/config/initializers/1_settings.rb

```diff
@@ -0,0 +1,8 @@
+class Settings < Settingslogic
+  source "#{Rails.root}/config/air.yml"
+  namespace Rails.env
+
+  Settings['sidekiq'] ||= Settingslogic.new({})
+  Settings.sidekiq['url'] ||= "redis://localhost:6379"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.